ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -277,3 +277,47 @@ def test_hamming_window(
         beta=beta,
         dtype=dtype,
     )
+
+
+@handle_test(
+    fn_tree="functional.experimental.tril_indices",
+    n_rows=helpers.ints(min_value=0, max_value=10),
+    n_cols=st.none() | helpers.ints(min_value=0, max_value=10),
+    k=helpers.ints(min_value=-11, max_value=11),
+    dtypes_x=helpers.dtype_and_values(available_dtypes=("int32", "int64")),
```

Comment:
To make this more efficient, we should directly use `helpers.get_dtypes()` with `full=False`, so we do not generate an array.